relations/jf/src/deposit.rs

```diff
@@ -1,10 +1,9 @@
-use ark_ff::Zero;
-use jf_primitives::circuit::rescue::RescueNativeGadget;
-use jf_relation::{Circuit, PlonkCircuit};
+use jf_relation::{Circuit, PlonkCircuit, Variable};
 
 use crate::{
+    note::{NoteRelation, NoteType},
     shielder_types::{convert_hash, Note, Nullifier, TokenAmount, TokenId, Trapdoor},
-    CircuitField, Marshall, PlonkResult, Relation,
+    CircuitField, PlonkResult, ProofSystem, PublicInput, Relation,
 };
 
 #[derive(Copy, Clone, Eq, PartialEq, Hash, Default, Debug)]
```

Comment:
actually, we need just a single field: `NoteRelation<Deposit>`

Reply:
changed